backend_new/src/dtos/listings/listings-query-params.dto.ts

```diff
@@ -0,0 +1,93 @@
+import { PaginationAllowsAllQueryParams } from '../shared/pagination.dto';
+import { Expose, Type } from 'class-transformer';
+import { ApiProperty, getSchemaPath } from '@nestjs/swagger';
+import { ListingFilterParams } from './listings-filter-params.dto';
+import {
+  ArrayMaxSize,
+  IsArray,
+  IsEnum,
+  IsString,
+  MinLength,
+  Validate,
+  ValidateNested,
+} from 'class-validator';
+import { ValidationsGroupsEnum } from '../../enums/shared/validation-groups-enum';
+import { ListingOrderByKeys } from '../../enums/listings/order-by-enum';
+import { OrderByEnum } from '../../enums/shared/order-by-enum';
+import { OrderQueryParamValidator } from '../../utilities/order-by-validator';
+
+export class ListingsQueryParams extends PaginationAllowsAllQueryParams {
+  @Expose()
+  @ApiProperty({
+    name: 'filter',
+    required: false,
+    type: [String],
+    items: {
+      $ref: getSchemaPath(ListingFilterParams),
+    },
+    example: { $comparison: '=', status: 'active', name: 'Coliseum' },
```

Comment:
so I ended up dropping the name: "Coliseum" since now instead of sending 1 object with all the = comparisons a person would multiple = comparison filters 1 for each

so like in that example it would be 
{ $comparison: '=', status: 'active' }
{ $comparison: '=', name: 'Coliseum' }